packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -2344,7 +2344,7 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin, Resto
   ///
   ///   * [ScaffoldMessenger], this should be used instead to manage [SnackBar]s.
   @Deprecated(
-    'Use ScaffoldMessenger.hideCurrentSnackBar. '
+    'Use ScaffoldMessenger.of(context).hideCurrentSnackBar'
```

Comment:
Hey @ScottS2017, I think this is a misunderstanding. The deprecation message here is intended to refer to `class.methodName`, not to provide actual runnable code.